packages/api/src/controllers/access-control.ts

```diff
@@ -23,7 +23,7 @@ import { isFreeTierUser } from "./helpers";
 import { cache } from "../store/cache";
 
 const WEBHOOK_TIMEOUT = 30 * 1000;
-const MAX_ALLOWED_VIEWERS_FOR_FREE_TIER = 30;
+const MAX_ALLOWED_VIEWERS_FOR_FREE_TIER = 5;
```

Comment:
I'd change the variable name to `MAX_ALLOWED_VIEWERS_FOR_HACKER_TIER_PER_NODE` or comment that it's per-node value. It's not obvious in this context.